src/modules/user/infra/http/controllers/UserController.ts

```diff
@@ -1,12 +1,10 @@
-import { Request, Response } from 'express';
-
-import { container } from 'tsyringe';
-
 import CreateUserService from '@modules/user/services/CreateUserService';
 import DeleteUserService from '@modules/user/services/DeleteUserService';
+import ListUserService from '@modules/user/services/IndexUserService';
 import ShowProfileService from '@modules/user/services/ShowProfileService';
 import UpdateProfileService from '@modules/user/services/UpdateProfileService';
-import ListUserService from '@modules/user/services/IndexUserService';
+import { Request, Response } from 'express';
+import { container } from 'tsyringe';
 
```

Comment:
We have to add a user show, we only have one for the user who is logged in the system, so as not to give the controller a problem, as there are two show methods on the controller, you can create a new controller called ProfileController and put the show current method on it.